### PR TITLE
fix spelling mistakes

### PR DESCRIPTION
**Pull request by BenjaminW3**
A few more spelling mistakes...

### CODE REVIEW

**Comment by ax3l** [comment]
I'll take that :)

File: src/libPMacc/include/math/vector/Vector.hpp [hunk 1]
```diff
@@ -188,9 +188,9 @@ struct Vector : private T_Storage<T_Type, T_dim>, protected T_Accessor, protecte
     }
 
     /**
-     * Creates a Vector where all dimensions are set to the same value
+     * Give a Vector<...> were all dimensions were set to the same value
```
Comment (ax3l) [inline src/libPMacc/include/math/vector/Vector.hpp]: "Give"? Rly?

Imho, "Return" or "Create" would be better.

File: src/picongpu/include/simulation_defines/param/gridConfig.param [hunk 2]
```diff
@@ -48,7 +48,7 @@ namespace picongpu
         {32, 32}   /*z direction [negative,positive]*/
     }; //unit: number of cells
 
-    //! Define the strange of the absober for any direction
+    //! Define the strength of the absorber for any direction
```
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gridConfig.param]: there are seven more `gridConfig.param` files in `examples/`. Do you mind moving that change to a separate commit and adding those?

File: src/libPMacc/include/math/vector/Vector.hpp (hunk 1 above)
Comment (BenjaminW3) [inline src/libPMacc/include/math/vector/Vector.hpp]: This was a merge error because I had previously fixed this but merged in the old version.

**Comment by BenjaminW3** [comment]
Should be fixed now.
The gridConfig.param changes have been proposed in an independent pull request.

**Comment by ax3l** [comment]
thx! excellent!
